src/main/java/core/basesyntax/FileWork.java

```diff
@@ -8,7 +17,31 @@
  * Результат: web wide width world</p>
  */
 public class FileWork {
+    private static final String CONSTANT = "w";
+
     public String[] readFromFile(String fileName) {
-        return null;
+        List<String> resultList = new ArrayList<>();
+
+        try {
+            File file = new File(fileName);
+            BufferedReader bufferedReader = new BufferedReader(new FileReader(file));
```

Comment:
readers should be closed after we don't need them. there is a method called `close()`. but to close them automatically, we should use try with resources